features/font-metric-overrides.yml

```diff
@@ -0,0 +1,11 @@
+name: Font metric overrides
+description: The `ascent-override`, `descent-override` and `line-gap-override` CSS properties adjust the display of a font declared in an `@font-face`. This can help make fallback fonts look the same size.
```

Comment:
```suggestion
description: The `ascent-override`, `descent-override` and `line-gap-override` descriptors adjust the display of a font declared in an `@font-face`. This can help make fallback fonts look the same size.
```